entries/a/app.ramp.com.json

```
@@ -0,0 +1,12 @@
+{
+  "Ramp": {
+    "domain": "app.ramp.com",
+    "additional-domains": ["ramp.com", "demo.ramp.com"],
+    "tfa": ["sms", "totp"],
```

If the entry adheres to the code styling that's used on all other entries it's much easier to read and compare.
```suggestion
    "tfa": [
      "sms",
      "totp"
    ],
```